### PR TITLE
Releasing v0.8.1

### PR DESCRIPTION
**Pull request by Vincent056**
This is for doing a downstream only SPO v0.8.1 release

### CODE REVIEW

File: hack/deploy-localhost.patch [hunk 1]
```diff
@@ -15,7 +15,7 @@ index d7bf1964..d2a9c614 100644
                fieldPath: spec.nodeName
          - name: KUBELET_DIR
            value: /var/lib/kubelet
--        image: registry.k8s.io/security-profiles-operator/security-profiles-operator:v0.8.0
+-        image: registry.k8s.io/security-profiles-operator/security-profiles-operator:v0.8.1
 -        imagePullPolicy: Always
 +        image: localhost/security-profiles-operator:latest
```
Comment (rhmdnd) [inline hack/deploy-localhost.patch:20]: Interesting - is this specific to the release?

Comment (saschagrunert) [reply hack/deploy-localhost.patch:20]: No, it's a patch used for e2e tests and local deployments.

**Comment by sheriff-rh** [comment]
lgtm, thanks!

/label docs-approved

File: hack/deploy-localhost.patch (hunk 1 above)
Comment (rhmdnd) [reply hack/deploy-localhost.patch:20]: Thanks @saschagrunert.

I imagine this should be removed from the PR, then.

Comment (Vincent056) [reply hack/deploy-localhost.patch:20]: sorry accidentally slipped that one in

Comment (Vincent056) [reply hack/deploy-localhost.patch]: @rhmdnd it should be updated now, thanks

**Review by rhmdnd — APPROVED** [review]
/lgtm

**Pull request merged**